HACKING.md

```diff
@@ -1,23 +1,30 @@
 # Hacking on charmhelpers
 
-
 ## Run testsuite
 
     make test
 
 Run `make` without arguments for more options.
 
-## Use it in a charm
+## Test it in a charm
 
-Use following instructions to build a charm that uses your own branch of
+Use following instructions to build a charm that uses your own development branch of
 charmhelpers.
 
-Create an override file `override-wheelhouse.txt` that points to your own
-charmhelpers branch.
+1) Make sure your version of charmhelpers is recognised as the latest version by
+by appending `dev0` to the version number in the `VERSION` file.
+
+2) Create an override file `override-wheelhouse.txt` that points to your own
+charmhelpers branch. *The format of this file is the same as pip's
+[`requirements.txt`](https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format)
+file.
 
-    -e git+https://github.com/<myuser>/charm-helpers#egg=charmhelpers
+    # Override charmhelpers by the version found in folder
+    -e /path/to/charmhelpers
+    # Or point it to a github repo with
+    # -e git+https://github.com/<myuser>/charm-helpers#egg=charmhelpers
 
-Build the charm specifying the override file. *You might need to install the
+3) Build the charm specifying the override file. *You might need to install the
 candidate channel of the charm snap*
 
     charm build <mycharm> -w wheelhouse-overrides.txt
```

Comment:
I recommend wrapping this with ``` to denote it as code:

```
charm build <mycharm> -w wheelhouse-overrides.txt
```

Reply:
4 line indent makes it a code block too though

Reply:
Sorry, it's messed up because of the bullet, either way, 4 more indents or tick fence

Reply:
Good catch! It looked fine in the markdown viewer of atom. I'll fix it.